dbt/adapters/snowflake/connections.py

```diff
@@ -48,7 +48,11 @@
 
 logger = AdapterLogger("Snowflake")
 _TOKEN_REQUEST_URL = "https://{}.snowflakecomputing.com/oauth/token-request"
-ROW_VALUE_REGEX = re.compile(r"Row Values: \[(.|\n)*\]")
+
+ERROR_REDACTION_PATTERNS = {
```

Comment:
I like that this is more configurable. Have you considered swapping the keys and the values? I could see adding to this in the future with more patterns; but some of those patterns may have the same replacement message, which wouldn't be possible with this setup.